src/test/jtx/Env.h

```diff
@@ -703,7 +740,9 @@ Env::rpc(
     Args&&... args)
 {
     return do_rpc(
-        std::vector<std::string>{cmd, std::forward<Args>(args)...}, headers);
+        std::vector<std::string>{cmd, std::forward<Args>(args)...},
+        RPC::apiMinimumSupportedVersion,
+        headers);
```

Comment:
Nope, that's a design mistake that took me whole day to unwind, and only partially. We want all unit tests for command line to pass `forAllApiVersions` and, when this happens, remove this default entirely.

In other words, we want to be able to switch `apiCommandLineVersion` which you proposed at will, knowing that that the newly selected (and any other) version is already covered by unit tests which do pass, before we even make such change.

Reply:
Huh, there are too many tests to use `forAllApiVersions` everywhere. So majority of tests use exactly the same version as the one used by `RPCCall` i.e. `apiCommandLineVersion` - this will give us the minimum of test coverage I was after. I will update this line accordingly.